src/OMSimulatorLib/TLM/SystemTLM.h

```diff
@@ -112,6 +113,7 @@ namespace oms
     double logStep = 1e-2;
     std::map<TLMBusConnector*, int> busLogIds;
     std::map<Connector*, int> connectorLogIds;
+    std::string singleModel;
```

Comment:
What is `singleModel` used for?

Reply:
Interfaces are fetched by simulating the TLM model a single step. Since some submodels take much longer time to initialize than others, we don't want to always include all the submodels. With the `singleModel` argument, we instruct OMTLMSimulator to only include the specified submodel and no connections.